src/ace/managers/state.c

```diff
@@ -0,0 +1,136 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include <ace/managers/state.h>
+
+#include <ace/managers/log.h>
+
+/* Functions */
+tStateManager *stateManagerCreate() {
```

Comment:
in C, empty parameter list means "0 or 1 arguments", whereas in C++ it means "0 arguments". It' best to write `void` to strictly tell compiler that you mean 0 args.